doc/release-notes.md

```diff
@@ -130,25 +159,26 @@ cases it also reduces propagation delay. It is automatically enabled between
 compatible peers.
 [BIP 152](https://github.com/bitcoin/bips/blob/master/bip-0152.mediawiki)
 
-Hierarchical Deterministic Key Generation
------------------------------------------
-Newly created wallets will use hierarchical deterministic key generation
-according to BIP32 (keypath m/0'/0'/k').
+[Non-Hierarchical] Deterministic Key Generation
+-----------------------------------------------
+Newly created wallets will use deterministic key generation according to BIP32
+(non-hierarchical, with a keypath m/0'/0'/k').
 Existing wallets will still use traditional key generation.
 
-Backups of HD wallets, regardless of when they have been created, can
-therefore be used to re-generate all possible private keys, even the
+Backups of deterministic wallets, regardless of when they have been created,
+can therefore be used to re-generate all possible private keys, even the
 ones which haven't already been generated during the time of the backup.
 **Attention:** Encrypting the wallet will create a new seed which requires
 a new backup!
 
-HD key generation for new wallets can be disabled by `-usehd=0`. Keep in
-mind that this flag only has affect on newly created wallets.
-You can't disable HD key generation once you have created a HD wallet.
+Deterministic key generation for new wallets can be disabled by `-usehd=0`.
+Keep in mind that this flag only has affect on newly created wallets. You
+can't disable deterministic key generation once you have created a
+deterministic wallet.
 
 There is no distinction between internal (change) and external keys.
 
-HD wallets are incompatible with older versions of Bitcoin Core.
+Deterministic wallets are incompatible with older versions of Bitcoin Core.
```

Comment:
"HD" is the term for hierarchical deterministic wallets. I am making the observation that Core 0.13's is not really hierarchical. TBD if anyone agrees of I should back this change out.